en/docs/get-started/quick-start-guide.md

```diff
@@ -109,90 +101,87 @@ In this guide, we will add a new resource to the API and simple business logic:
 
      ![Configure Variable](../assets/img/get-started/configure-variable.png){.cInlineImage-full}
 
-9. Click **Save**.
+8. Click **Save**.
 
-10. Let's return the message. Click **+** below the variable we created on the low-code diagram and select **Return**. 
+9. To return the personalized message, click **+** below the variable you created,  click **Return**, and enter `message` as the **Return Expression**.
```

Comment:
There is a double- space after the comma?

Reply:
Fixed.